### DISCUSSION TITLE
Export conversation threads?

### DISCUSSION THREAD

**Discussion by ar5000** [plswiderski/asanaexporter#20] [Q&A] [answered]
Sorry for posting this here.  I tried to contact you via your website, but the contact form just gave me an error.

I am downsizing my team, and will eventually transition away from Asana.  However, most of the juicy bits I want to keep are in the conversation threads.  Images, prices, hyperlinks, etc.  I'm trying to figure out a way to keep all of that.  Did you ever find a way to export more than just the name, date, and DRI of the tasks?  I thought I would ask before reinventing the wheel.

Thanks!

Alex

**Comment by plswiderski** [comment, marked as answer]
Hi, sorry for the delay in my answer.
As I checked currently Asana does not provide an API for getting conversations. It was mentioned on their forum: https://forum.asana.com/t/asana-conversations/42092 . They have it in their backlog so maybe in the future it will be possible.
When it comes to attachments it is possible to download them https://developers.asana.com/reference/getattachment

I hope I helped.
